src/shared/utils/getDotnetInfo.ts

```diff
@@ -70,7 +70,8 @@ async function parseDotnetInfo(dotnetInfo: string, dotnetExecutablePath: string
         }
 
         const runtimeVersions: { [runtime: string]: RuntimeInfo[] } = {};
-        const listRuntimes = await execChildProcess('dotnet --list-runtimes', process.cwd(), process.env);
+        const command = dotnetExecutablePath ? `"${dotnetExecutablePath}"` : 'dotnet';
+        const listRuntimes = await execChildProcess(`${command} --list-runtimes`, process.cwd(), process.env);
```

Comment:
This is most likely all correct, but give me a bit to pull this down locally and run it through an extra gamut of tests.  Changes here are always a bit dangerous.

Reply:
Thanks @dibarbet, I totally understand it and really appreciate it.